Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -0,0 +1,68 @@
+using Uno;
+using Uno.Collections;
+using Uno.Testing;
+using Fuse;
+using FuseTest;
+
+namespace Fuse.ImageTools.Test
+{
+	public class ImageToolsTest : TestBase
+	{
+		private Object _image = null;
+
+
+		[Test]
+		public void CreatingAnImageSucceeds()
+		{
+			if defined(!DOTNET) return;
+
+			Fuse.ImageTools.ImageTools image = new Fuse.ImageTools.ImageTools();
+			using (var root = new TestRootPanel()){
+				var future = ImageTools.ImageFromBase64("iVBORw0KGgoAAAANSUhEUgAAAA8AAAAPAQMAAAABGAcJAAAABlBMVEX9//wAAQATpOzaAAAAH0l" + "EQVQI12MAAoMHIFLAAYSEwIiJgYGZASrI38AAAwBamgM5VF7xgwAAAABJRU5ErkJggg==");
+				future.Then(Print, Fail);
+				root.StepFrame();
+				root.StepFrame();
+				root.StepFrame();
+				root.StepFrame();
```

Comment:
If you make a closure-object instead, we can step until we either fail, or pass here instead, so we don't need to know the timing...